app/src/app_config.py

```diff
@@ -44,3 +44,6 @@ def sentence_transformer(self) -> SentenceTransformer:
 
 
 app_config = AppConfig()
+
+app_config.sentence_transformer.tokenizer
+
```

Comment:
Stray code?
```suggestion
```

Reply:
Oh yikes, that'll teach me to not read through even "one line" PRs before submitting them!